source/aggregation-tutorials/multi-field-join.txt

```diff
@@ -4,4 +4,240 @@
+            values of two fields on each side of the join:
+
+            .. literalinclude:: /includes/aggregation/multi-field-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-embedded-pl-match1
+               :end-before: end-embedded-pl-match1
+               :dedent:
+
+            Within the embedded pipeline, add another :manual:`$match
+            </reference/operator/aggregation/match>` stage to match
+            orders placed in 2020:
+
+            .. literalinclude:: /includes/aggregation/multi-field-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-embedded-pl-match2
+               :end-before: end-embedded-pl-match2
+               :dedent:
+
+            Within the embedded pipeline, add an :manual:`$unset
+            </reference/operator/aggregation/unset>` stage to remove
+            unneeded fields from the ``orders`` collection side of the join:
+
+            .. literalinclude:: /includes/aggregation/multi-field-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-embedded-pl-unset
+               :end-before: end-embedded-pl-unset
+               :dedent:
+
+            After the embedded pipeline is completed, add the
+            ``$lookup`` stage to the main aggregation pipeline that also stores
+            the processed lookup fields in the ``orders`` array field:
+
+            .. literalinclude:: /includes/aggregation/multi-field-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-lookup
+               :end-before: end-lookup
+               :dedent:
+
+         .. step::
+
+            Next, add a :manual:`$match
+            </reference/operator/aggregation/match>` stage to only show
+            products for which there is at least one order in 2020,
+            based on the ``orders`` array calculated in the previous step:
+
+            .. literalinclude:: /includes/aggregation/multi-field-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-match
+               :end-before: end-match
+               :dedent:
+
+         .. step::
+
+            Finally, add an :manual:`$unset
+            </reference/operator/aggregation/unset>` stage. The
+            ``$unset`` stage removes the ``_id`` field from the result documents:
+            
+            .. literalinclude:: /includes/aggregation/multi-field-join.js
+               :language: javascript
+               :copyable: true
+               :start-after: start-unset
+               :end-before: end-unset
+               :dedent:
+
+   .. step:: Run the Aggregation Pipeline
+
+      Perform the aggregation on the ``products`` collection by running the
+      following command in your shell to run the ``agg_tutorial.js``
+      file:
```

Comment:
S: Not necessary, but I found the current wording slightly confusing
```suggestion
      Run the `agg_tutorial.js`` file to perform the aggregation on the ``products`` collection by running the
      following command in your shell:
```

Reply:
Simplified and changed in both existing tutorials